workers-docs/src/content/tooling/wrangler/commands.md

```diff
@@ -164,6 +164,31 @@ If you would like to be able to publish your code to multiple places, please see
    * `--port PORT`: the port for your local log server
    * `--metrics-port PORT`: the port for serving [metrics information](https://developers.cloudflare.com/argo-tunnel/reference/arguments/#metrics) about the tunnel.
 
+#### Parsing with jq
+
+  tail outputs each request as a JSON object, making it difficult to read directly. Parsing with jq makes it easy to pretty-print with colorization as well as select and format output as needed.
+
+  For example, this jq command will flatten console.log() statements:
+
+  ```shell
+  wrangler tail | jq -r \
+       '# loop through all logs
+    .logs[] as $o |
```

Comment:
can the `$o` variable be changed to `$log` here? i think it's a more descriptive variable name :)

Reply:
Great point! That was a leftover from when this was a one-liner made as short as possible. I've pushed a commit clarifying variable names.